app/index.html

```diff
@@ -8,9 +8,11 @@
 
     <link rel="stylesheet" href="css/windowing.css">
   </head>
-<iframe src="home.html"></iframe>
+<body>
+
```

Comment:
There might have been but I can't remember so I moved that back into `<body>`. And it works.